README.md

```diff
@@ -224,6 +226,27 @@ profile](https://github.com/solid/solid-spec/blob/master/solid-webid-profiles.md
 the profile document itself, any `sameAs` and `seeAlso` links it finds there,
 as well as the Preferences file.
 
+Once a profile is loaded, you can access the values of the profile's pre-defined
+fields, or look for predicates in the profile's parsed graph using 
+`profile.find()` and `profile.findAll()`:
+
+```js
+var vocab = solid.vocab
+solid.login()
+  .then(function (webId) {
+    return solid.getProfile(webId)
+  })
+  .then(function (profile) {
+    profile.name  // -> 'Alice'
+    profile.img   // -> 'https://example.com/profile/icon.png'
+    profile.find(vocab.solid('inbox'))    // -> 'https://example.com/inbox/'
+    profile.findAll(vocab.owl('sameAs'))  // -> [ url1, url2 ]
+  })
+  .catch(function (err) {
+    console.log('Error accessing profile: ' + err)
```

Comment:
I recently learned that `console.log` will concatenate its arguments (with spacing!) for you; so `console.log('foo:', err)` is equivalent to `console.log('foo: ' + err)`